### DISCUSSION TITLE
How to modify the config after its setup and running?

### DISCUSSION THREAD

**Discussion by balajeek** [marcone/teslausb#793] [Q&A]
I setup the teslausb and its been running for weeks, copying files into my nas when i get home.
I see it copies only "Saved" and "Sentry" folders with files everyday. "Recent" folder is empty.

What should i do if i also need to copy the "Recent" which have all the recordings of my driving.

If there is such option to copy then how would i access the config and make the changes?
by SSH or remove the SD card and access it ?

Please let me know.  Thanks.

**Comment by marcone** [comment]
https://github.com/marcone/teslausb/wiki/FAQ#q-how-do-i-change-the-teslausb-configuration

**Reply by balajeek** [reply]
thanks for the link, does this mean i can SSH and modify or remove the SD card and modify?
Also what do i modify?  Would be helpful, i have been reading documentation over and over again and i can't seem to find my answer.

**Reply by marcone** [reply]
You ssh into the Pi, and edit /root/teslausb_setup_variables.conf
There's a setting in there to archive RecentClips

**Comment by balajeek** [comment]
Thank you, will try tonight when i get home.

**Comment by balajeek** [comment]
Sorry for the delay, I tried but unable to modify the conf file becaz of permission denied

I did sudo -i
then sudo nano teslausb_setup_variables.conf and i get following error:

```
Unable to create directory /root/.local/share/nano/: No such file or directory
It is required for saving/loading search history or cursor positions.
Press Enter to continue
```

and continue and uncomment the recent clips options, but when i try saving it give me error:

`[ File 'teslausb_setup_variables.conf' is unwritable ]`

**Reply by THX723** [reply]
It appears you omitted this part...

"To make the root partition writeable so you can make changes, run /root/bin/remountfs_rw first."

**Comment by marcone** [comment]
See the link I sent earlier:
![image](https://github.com/marcone/teslausb/assets/48169102/4de6d546-bbeb-4c66-a0db-f43dd31a6b05)
